yarn-project/aztec/package.json

```diff
@@ -17,6 +17,7 @@
     "build": "yarn clean && tsc -b",
     "start": "node --no-warnings ./dest/bin",
     "start:debug": "node --inspect=0.0.0.0:9221 --no-warnings ./dest/bin",
+    "start:sandbox": "ETHEREUM_HOST=http://0.0.0.0:8545/ && yarn start start --sandbox",
```

Comment:
Sneaked this in because of [this discussion](https://aztecprotocol.slack.com/archives/C02M7VC7TN0/p1735829107540819) on slack.